submit.py

```diff
@@ -370,12 +366,7 @@ def show_judgement(submission_url, cfg):
             success = status_id == _ACCEPTED_STATUS
             if success:
                 status_text += f' ({score})'
```

Comment:
Something like
```py
            if success:
                status_text += f' ({score}/{maxscore})'
```
would be nice. 

Is it possible to include the `maxscore` information from Kattis?

Reply:
Unfortunately, there's no `maxscore` information available from/in Kattis, so there's no way to correctly show that (without making assumptions, which could be incorrect...)